### PR TITLE
#171 Intro

### PR DESCRIPTION
**Pull request by AnushkaS77**
- **Title**:  intro at begining 
- **Your Name**: AnushkaS77
- **Idenitfy yourself**: gssoc contributer

### Describe the add-ons or changes you've made 📃
made changes in home.py
added intro.gif file to assests 

### Checklist: ☑️

- [x ] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x ] This PR does not contain plagiarized content.
- [x ] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly wherever it was hard to understand.

### CODE REVIEW

**Comment by AnushkaS77** [comment]
[home.webm](https://github.com/user-attachments/assets/30461f8c-8b60-4b85-a2a3-02eb25ea20c7)

File: .streamlit/secrets.example.toml
Comment (Avdhesh-Varshney) [inline .streamlit/secrets.example.toml:1]: Undo these changes

File: requirements.txt
Comment (Avdhesh-Varshney) [inline requirements.txt:1]: Where have you use this package? If not required, remove it.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Do these changes, I'm providing the solution

**Comment by Avdhesh-Varshney** [comment]
Also move your `intro` video in `assets` folder

**Comment by AnushkaS77** [comment]
@Avdhesh-Varshney  if the file is in gif form ...it gets stuck 
image

**Comment by AnushkaS77** [comment]
@Avdhesh-Varshney ive done the changes you asked me to

**Review by Zedoman — APPROVED** [review]
